internal/scheduler/context/context.go

```diff
@@ -718,9 +754,8 @@ type PodSchedulingContext struct {
 	NodeId string
 	// Score indicates how well the pod fits on the selected node.
 	Score int
-	// Priority class priority at which this pod was scheduled.
-	// Only set if NodeId is.
-	ScheduledAtPriority int32
+	// Maximum priority that this pod preempted other pods at.
+	PreemptedAtPriority int32
```

Comment:
I'm not too fond of this name, because it's not immediately clear (at least without sufficient context) whether this means that this pod was preempted or that it preempted other pods, but I couldn't think of anything better.